policy/datasets.yaml

```diff
@@ -1501,6 +1501,20 @@ plotPhotonTransferCurve_metadata:
     storage: YamlStorage
     python: lsst.daf.base.PropertySet
     template: ''
+measureLinearity_config:
+    description: "Configuration for the linearity fitting task."
+    persistable: Config
+    python: lsst.cp.pipe.linearity.MeasureLinearityConfig
+    storage: ConfigStorage
+    tables:
+    - 'raw'
```

Comment:
I could be wrong, but I don't think `tables` is needed/makes sense here (and the other config stores I looked at don't define it).

Reply:
I think this is based (in some roundabout way) on other entries.  I've removed the `tables` attribute, and assume this won't cause issues.  I believe this is less fussy in gen3.